### ISSUE TITLE
Notification not showing up for me

### ISSUE THREAD

**Issue by florianluediger** [vaskovsky/notify-send#1]
When starting the application the notification does not show up. The tray icon appears and stays there until I click it but the notification does not appear, no matter how long I wait.

I am using Windows 10 x64

**Comment by vaskovsky** [comment]
The program has been tested on Windows 10 x64.
The problem is not detected.
![screenshot](https://cloud.githubusercontent.com/assets/12381353/17475676/260a6a44-5d6d-11e6-9603-15e41beb6665.png)
1. Did you use a [precompiled executable](http://vaskovsky.net/notify-send/notify-send.zip)?
   If not, which version of Visual Studio has been used to compile the program?
2. Run the program at the command prompt and write the command line parameters and program output.

**Comment by florianluediger** [comment]
I have used your precompiled exe and also tested to compile the project with Visual Studio Community 2015 and both get me the same behavior.

When I run the program from the console, there is no output at all. The tray icon appears and that's it.

I tried the following command for example:
`notify-send "Test" "Hello World!"`

I noticed however that after some time (approximately one or two minutes) after running the program, the notification pops up. It seems that something is holding the notification back for some time before displaying it. 

Also: The tray icon does not disappear, even after displaying the notification, until I klick on it.
